EIPS/eip-2980.md

```diff
@@ -0,0 +1,98 @@
+---
+eip: 2980
+title: Swiss Compliant Asset Token
+author: Gianluca Perletti (@Perlets9), Alan Scarpellini (@alanscarpellini), Roberto Gorini (@robertogorini), Manuel Olivi (@manvel79)
+discussions-to: https://github.com/ethereum/EIPs/issues/2983
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-08
+requires: 20
+---
+
+## Simple Summary
+
+An interface for asset tokens, compliant with Swiss Law and compatible with [ERC-20](./eip-20.md).
+
+## Abstract
+
+This new standard is an [ERC-20](./eip-20.md) compatible token with restrictions that comply with one or more one the following Swiss laws: the [Stock Exchange Act](./assets/eip-2980/Swiss-Confederation-SESTA.pdf), the [Banking Act](./assets/eip-2980/Swiss-Confederation-BA.pdf), the [Financial Market Infrastructure Act](./assets/eip-2980/Swiss-Confederation-FMIA.pdf), the [Act on Collective Investment Schemes](./assets/eip-2980/Swiss-Confederation-CISA.pdf) and the [Anti-Money Laundering Act](./assets/eip-2980/Swiss-Confederation-AMLA.pdf). The [Financial Services Act](./assets/eip-2980/Swiss-Confederation-FINSA.pdf) and the [Financial Institutions Act](./assets/eip-2980/Swiss-Confederation-FINIA.pdf) must also be considered. The solution achieved meet also the European jurisdiction.
```

Comment:
```suggestion
This new standard is an [ERC-20](./eip-20.md) compatible token with restrictions that comply with one or more one the following Swiss laws: the [Stock Exchange Act](../assets/eip-2980/Swiss-Confederation-SESTA.pdf), the [Banking Act](./assets/eip-2980/Swiss-Confederation-BA.pdf), the [Financial Market Infrastructure Act](../assets/eip-2980/Swiss-Confederation-FMIA.pdf), the [Act on Collective Investment Schemes](../assets/eip-2980/Swiss-Confederation-CISA.pdf) and the [Anti-Money Laundering Act](../assets/eip-2980/Swiss-Confederation-AMLA.pdf). The [Financial Services Act](../assets/eip-2980/Swiss-Confederation-FINSA.pdf) and the [Financial Institutions Act](../assets/eip-2980/Swiss-Confederation-FINIA.pdf) must also be considered. The solution achieved meet also the European jurisdiction.
```
The assets directory is in the parent folder, so you need `(../assets/eip-2980/...)`